packages/config/src/config/build.js

```diff
@@ -14,12 +14,12 @@ export default () => ({
   serverURLPolyfill: 'url',
   filenames: {
     // { isDev, isClient, isServer }
-    app: ({ isDev, isModern }) => isDev ? `${isModern ? 'modern-' : ''}[name].js` : '[contenthash].js',
-    chunk: ({ isDev, isModern }) => isDev ? `${isModern ? 'modern-' : ''}[name].js` : '[contenthash].js',
-    css: ({ isDev }) => isDev ? '[name].css' : '[contenthash].css',
-    img: ({ isDev }) => isDev ? '[path][name].[ext]' : 'img/[contenthash:7].[ext]',
-    font: ({ isDev }) => isDev ? '[path][name].[ext]' : 'fonts/[contenthash:7].[ext]',
-    video: ({ isDev }) => isDev ? '[path][name].[ext]' : 'videos/[contenthash:7].[ext]'
+    app: ({ isDev, isModern }) => isDev ? `${isModern ? 'modern-' : ''}[name].js` : '[name].[contenthash:7].js',
+    chunk: ({ isDev, isModern }) => isDev ? `${isModern ? 'modern-' : ''}[name].js` : '[name].[contenthash:7].js',
+    css: ({ isDev }) => isDev ? '[name].css' : '[name].[contenthash:7].css',
```

Comment:
IIRC we discussed using `name` for css, images etc. again but back then it was classified as breaking change.

Reply:
Actually it was mostly to not exposing anything from source. But the fact that already even component names can be reversed, it is just making debugging harder IMO. But have no objections using name or not for assets ☺️

Reply:
Not using it for js files is understandable for me, but img/css/fonts/videos/... always felt a bit wrong, especially bc image names can help in various places.

Reply:
Exactly. Also probably can help for CEO (searching images without alt)

I will keep change as is and comments unresolved if there is anything potentially breaking.

Reply:
Yup, SEO is one of the things where this comes in handy. Mentioned that in my previous talks mostly (+ how to change in the nuxt config).

Great 👍🏻